src/unit_tests/ctx/update_streaming_server_urls.rs

```diff
@@ -0,0 +1,111 @@
+use crate::constants::STREAMING_SERVER_URLS_STORAGE_KEY;
+use crate::models::ctx::Ctx;
+use crate::runtime::msg::{Action, ActionCtx};
+use crate::runtime::{Env, Runtime, RuntimeAction};
+use crate::types::events::DismissedEventsBucket;
+use crate::types::library::LibraryBucket;
+use crate::types::notifications::NotificationsBucket;
+use crate::types::profile::Profile;
+use crate::types::search_history::SearchHistoryBucket;
+use crate::types::server_urls::ServerUrlsBucket;
+use crate::types::streams::StreamsBucket;
+use crate::unit_tests::{TestEnv, STORAGE};
+use stremio_derive::Model;
+use url::Url;
+
+#[test]
+fn test_add_server_url() {
```

Comment:
- [ ] Please add a test for the max number of items and the replacement (or not) of the oldest url

Reply:
Not needed since the logic will be removed and we will allow users to place new URLs without constrains